package.json

```diff
@@ -63,19 +63,18 @@
     "@semantic-release/github": "^7.1.1",
     "@semantic-release/npm": "^7.0.6",
     "@semantic-release/release-notes-generator": "^9.0.1",
-    "@storybook/addon-actions": "^6.4.14",
-    "@storybook/addon-backgrounds": "^6.4.14",
-    "@storybook/addon-links": "^6.4.14",
-    "@storybook/addon-options": "^5.3.21",
-    "@storybook/addons": "^6.4.14",
-    "@storybook/api": "^6.4.14",
-    "@storybook/channels": "^6.4.14",
-    "@storybook/client-logger": "^6.4.14",
-    "@storybook/core-events": "^6.4.14",
-    "@storybook/react": "^6.4.14",
-    "@storybook/router": "^6.4.14",
-    "@storybook/storybook-deployer": "^2.8.10",
-    "@storybook/theming": "^6.4.14",
+    "@storybook/addon-a11y": "^6.5.9",
+    "@storybook/addon-essentials": "^6.5.9",
+    "@storybook/addon-links": "^6.5.9",
+    "@storybook/addons": "^6.5.9",
+    "@storybook/api": "^6.5.9",
+    "@storybook/channels": "^6.5.9",
+    "@storybook/client-logger": "^6.5.9",
+    "@storybook/core-events": "^6.5.9",
+    "@storybook/react": "^6.5.9",
+    "@storybook/router": "^6.5.9",
```

Comment:
Is there a need to use this package in the project?

Reply:
I'll check this